src/service/vsync-impl.js

```diff
@@ -309,6 +334,14 @@ export class Vsync {
         tasks[i].mutate(states[i]);
       }
     }
+    // Swap last arrays into double buffer.
+    this.nextTasks_ = tasks;
```

Comment:
I don't understand the point of having `nextTasks_` if we just reassign to `tasks` and truncate.